includes/classes/Rest/IssuesAPI.php

```
@@ -0,0 +1,615 @@
+		foreach ( array_keys( $update_data ) as $field_name ) {
+			$set_clauses[] = '`' . esc_sql( $field_name ) . '` = ' . $allowed_fields[ $field_name ];
+		}
+		$set_sql = implode( ', ', $set_clauses );
+
+		$query_values = array_values( $update_data );
+		$query_values[] = $id; // For the WHERE clause.
+
+		// phpcs:disable WordPress.DB.DirectDatabaseQuery.DirectQuery, WordPress.DB.PreparedSQL.NotPrepared -- Using $wpdb->prepare correctly
+		$updated = $wpdb->query(
+			$wpdb->prepare(
+				"UPDATE `{$this->table_name}` SET {$set_sql} WHERE `id` = %d",
```

PHPStan (2.1.15)

Access to an undefined property EqualizeDigital\AccessibilityChecker\Rest\Issues_API::$table_name.
Learn more: https://phpstan.org/blog/solving-phpstan-access-to-undefined-property

(property.notFound)